pvm/ts/src/testing/phhParser.ts

```
@@ -206,8 +225,10 @@ export class PhhParser {
                     return { type: "fold", player, raw };
 
                 case "cc":
-                    // Check if there's a bet to call
-                    const type: PhhActionType = currentBet > 0 ? "call" : "check";
+                    // Check if this player needs to call or can check
+                    // Player can check if they've already matched the current bet
+                    const playerBet = playerBets.get(player) || 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)